perma_web/frontend/components/CreateLink.vue

```diff
@@ -158,7 +150,6 @@ const handleProgressUpdate = async () => {
     if (status === 'completed') {
         clearInterval(progressInterval)
         globalStore.updateCapture('success')
-        window.location.href = `${window.location.origin}/${userLinkGUID.value}`
```

Comment:
This has been moved to a watch function, so that it be triggered by both successful captures and successful uploads